Dockerfile

```diff
@@ -2,16 +2,22 @@
 
 FROM centos:centos7
 
-#
+LABEL maintainer="arXiv <nextgen@arxiv.org>" \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="arXiv Base" \
+  org.label-schema.description="Base image for arXiv NG applications" \
+  org.label-schema.url="https://arxiv.github.io/arxiv-base" \
+  org.label-schema.vcs-url="https://github.com/arxiv/arxiv-base" \
+  org.label-schema.vendor="arXiv.org"
```

Comment:
I am not too picky about this, but the [label-schema](http://label-schema.org/rc1/) is now deprecated in favor of [OCI Image Spec](https://github.com/opencontainers/image-spec/blob/master/annotations.md).

Reply:
Sounds good; switched in 67927e7